src/Network/Network/help/Add-AzExpressRouteCircuitConnectionConfig.md

```diff
@@ -16,21 +16,21 @@ Adds a circuit connection configuration to Private Peering of an Express Route C
 ### SetByResource (Default)
 ```
 Add-AzExpressRouteCircuitConnectionConfig [-Name] <String> [-ExpressRouteCircuit] <PSExpressRouteCircuit>
- [-AddressPrefix] <String> [-AuthorizationKey <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
+ [-AddressPrefix] <String> -AddressPrefixType <String> [-AuthorizationKey <String>] [-DefaultProfile <IAzureContextContainer>] [-WhatIf]
```

Comment:
This is expected to be optional, i.e. `[-AddressPrefixType <String>]`. Please regenerate the document and see if it's right?

Reply:
Thanks. Done.
By the way, the auto-regeneration didn't work for me and my other teams mates as well. 
Who can we discuss the same with ?